modules/builds-buildconfig.adoc

```diff
@@ -54,7 +54,7 @@ spec:
 ifndef::openshift-online[]
 `Dockerfile`, to build from an inline Dockerfile,
 endif::[]
-or `Binary`, to accept binary payloads. It is possible to have multiple sources at once. See the documentation for each source type for details.
+or `Binary`, to accept binary payloads. It is possible to have multiple sources at once. For more information on each source type, see "Creating build inputs".
```

Comment:
```suggestion
or `Binary`, to accept binary payloads. It is possible to have multiple sources at once. For more information about each source type, see "Creating build inputs".
```

Reply:
Also, while you are here, please add a content type attribute in the metadata: `:_content-type: <TYPE>`